community/deb.sh

```diff
@@ -249,9 +258,15 @@ main ()
 
   echo -n "Installing $apt_source_path... "
 
-  # create an apt config file for this repository
   curl -sSf "${apt_config_url}" > $apt_source_path
   curl_exit_code=$?
+  if [ "${os}" = "debian" ] && [ "$curl_exit_code" -ne "0" ]; then
```

Comment:
Can we add a comment before this conditional to explain why we need such a change?